src/main/java/ecdar/abstractions/ComponentInstance.java

```diff
@@ -11,8 +9,8 @@
  * Instance of a component.
  */
 public class ComponentInstance implements SystemElement {
-    public final static int WIDTH = Grid.GRID_SIZE * 22;
-    public final static int HEIGHT = Grid.GRID_SIZE * 12;
+    public final static int WIDTH = 220;
+    public final static int HEIGHT = 120;
```

Comment:
Consider making this a function of the canvas padding:
```suggestion
    public final static int WIDTH = Ecdar.CANVAS_PADDING * 22;
    public final static int HEIGHT = Ecdar.CANVAS_PADDING * 12;
```

Reply:
This introduces a dependency between the padding and component size, which can result in unforeseen side-effects if the padding is changed. There is no real connection between the padding and the component sizes IMO